### ISSUE TITLE
donation link

### ISSUE THREAD

**Issue by llj098** [orgzly/orgzly-android#673]
Request for the donation link. 

Thanks for the App

**Comment by gsunjka** [comment]
Yes, me too. Thanks a lot for this great app. Saves me tons of time. 🥇

**Comment by azzamsa** [comment]
related: https://github.com/orgzly/orgzly-android/issues/110

**Comment by stevenov** [comment]
Also interested in this feature.
